build.sbt

```diff
@@ -1,3 +1,5 @@
+import java.io.File
```

Comment:
`java.io.File` should be available as `sbt.File`.

Reply:
Will fix thanks 👍

Reply:
If I change to importing `sbt.File` I get ` not found: value File`. I tried removing the import, since I thought `sbt._` was supposed to be imported by default, but that had the same effect.

Reply:
https://github.com/sbt/sbt/blob/v1.1.1/sbt/src/main/scala/Import.scala#L14
Oh that's because we don't import `val` but only the `type`. I guess you do need the import then. My bad.